doc/zmq.md

```diff
@@ -1,6 +1,6 @@
 # Block and Transaction Broadcasting with ZeroMQ
 
-[ZeroMQ](http://zeromq.org/) is a lightweight wrapper around TCP
+[ZeroMQ](https://zeromq.org/) is a lightweight wrapper around TCP
```

Comment:
How are we stil finding http links in the repository in the year 2020 :sweat_smile:

Reply:
Thanks for fixing @jonatack! :)

A `scripted-diff` might be needed to bring the remaining old school style links to year 2020 :)

```
$ git grep 'http://www.opensource.org/licenses/mit-license.php' | wc -l
1092
```

Bonus points if we can remove the external -- and thus potentially changing -- reference to simply referencing the `COPYING` file. That would get rid of the horrible `php` extension too :)

Reply:
True :dango: :icecream:  though updating all those links isn't really in the scope of this patch. I actually only noticed the http url just before opening it. :shipit:

Reply:
Ahhhhh updating 1002 files sounds … pretty awful … that fully answers my question though. Together with XML schemas and such it makes it hard to search for relevant ones. 
Agree it's out of scope of this PR.